bindings/java/src/test/com/apple/foundationdb/test/StackTester.java

```diff
@@ -499,6 +499,11 @@ else if(op == StackOperation.LOG_STACK) {
 
 				logStack(inst.context.db, entries, prefix);
 			}
+			else if (op == StackOperation.GET_ESTIMATED_RANGE_SIZE) {
```

Comment:
A minor nit again, but this may sort more naturally above the unit tests.

Reply:
Done